### ISSUE TITLE
Tidy Up Refinery Pages

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#413]
Many pages moved over from FreeREG1 that will not be part of FreeREG2. The list of pages is quite confusing so we could delete a lot of them.

**Comment by DarrenWright** [comment]
Many pages moved over from FreeREG1 that will not be part of FreeREG2. The list of pages is quite confusing so we could delete a lot of them.

**Comment by AlOneill** [comment]
Have tidied up the 'Information for transcribers' folder a little: all the pages from the top down to 'Uploading files' have been updated for FR2; the remainder of the pages are FR1 and either in need of updating, or their updated content moved/covered somewhere else.  (Except for 'Glossary' which is new and has an uncertain future.)

OnlyTranscriber FR2 pages have links to them, so users without Refinery access will not be confused. 

From my point of view, the FR1 pages are still useful as pointers to what might need to be covered somewhere. However, I did delete 2 pages that were clearly not still needed ('Data fields' and one with a long name that has been superceded by 'Uploading files').
